CorsixTH/com.github.CorsixTH.CorsixTH.appdata.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>corsix-th.desktop</id>
```

Comment:
Isn't id suppose to be the reverse DNS name, like the name of the file? https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-id-generic and https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html#sect-Quickstart-DesktopApps sound that way.